src/bridge/jsHelpers.js

```diff
@@ -381,23 +381,36 @@ export function makeAccountBridgeReceive({
   injectGetAddressParams?: (Account) => *,
 } = {}): (
   account: Account,
-  { verify?: boolean, deviceId: string, subAccountId?: string }
+  {
+    verify?: boolean,
+    deviceId: string,
+    subAccountId?: string,
+    freshAddressIndex?: string,
+  }
 ) => Observable<{
   address: string,
   path: string,
 }> {
-  return (account, { verify, deviceId }) => {
+  const getFreshAddressByIndex = (account, index) => {
+    return account.freshAddresses[Number(index)];
```

Comment:
this could fail and we need a dedicated error to say "FreshAddressIndexInvalid" or something. because we don't really want `derivationPath of undefined` errors.